CHANGELOG.md

```diff
@@ -21,6 +21,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - `custom/text`: Loads the `format` setting, which supports the `<label>` tag, if the deprecated `content` is not defined ([`#1331`](https://github.com/polybar/polybar/issues/1331), [`#2673`](https://github.com/polybar/polybar/pull/2673), [`#2676`](https://github.com/polybar/polybar/pull/2676))
 - Added experimental support for positioning the tray like a module
 
+### Fixed
+- Waiting for double click interval on modules that don't have a double click action ([`#2695`](https://github.com/polybar/polybar/pull/2695))
```

Comment:
Please also add a link to the original issue https://github.com/polybar/polybar/issues/2663